server/src/github.com/alexeyknyshev/cpsrv/cpsrv_test.go

```diff
@@ -1485,3 +1487,87 @@ func TestTimeFilter(t *testing.T) {
 	}
 
 }
+
+//Test metro
+type Metro struct {
+	StationName     string  `json:"station_name"`
+	Longitude       float64 `json:"longitude"`
+	Latitude        float64 `json:"latitude"`
+	Id              uint32  `json:"id"`
+	StationExitName string  `json:"station_exit_name"`
+	TownId          uint32  `json:"town_id"`
+	BranchId        uint32  `json:"branch_id"`
+}
+
+func getMetroTuple() Metro {
+	expected := Metro{
+		StationName:     "Беляево",
+		Longitude:       37.526596999601,
+		Latitude:        55.643621500032,
+		Id:              779,
+		StationExitName: "Беляево, вход-выход 2 в северный вестибюль",
```

Comment:
As soon as **ext** field since now has not **name** in it we need to update testing data.